doc/api/stream.md

```diff
@@ -730,8 +730,7 @@ the [`'drain'`][] event before writing more data.
 
 To implement any sort of stream, the pattern is the same:
 
-1. Extend the appropriate parent class in your own subclass. (The
-   [`util.inherits()`][] method is particularly helpful for this.)
+1. Extend the appropriate parent class in your own subclass via ES6 `extends`.
```

Comment:
I'd write this as 'the `extends` keyword`, prefix with 'ES6' if desired, although I don't think that's necessary.